regex-assembly/942120.ra

```diff
@@ -31,7 +32,7 @@ like\s+null
 \bnotnull\b
 like\s+[\w]+\s+escape\b
 \bilike\b
-[<>=!]{1,2}\s*all\b
+[<>=!]\s*all\b
```

Comment:
In the spirit of not losing the meaning here (whatever it was), let's add two tests:
- one for the one char match
- another for the two chars match (that will match the above operators, but just for the sake of testing we comply with the meaning of the previous rule)

Reply:
Added tests.